easybuild/easyconfigs/a/AmberTools/AmberTools-21-foss-2021a.eb

```
@@ -11,18 +11,27 @@ description = """AmberTools consists of several independently developed packages
 toolchain = {'name': 'foss', 'version': '2021a'}
 toolchainopts = {'usempi': True}
 
-sources = [SOURCE_TAR_BZ2]
+# download requires registration, see http://ambermd.org/AmberTools17-get.html
+local_download_credentials = '?Name=Easybuild&Institution=Easybuild&City=Internet&State=Other&Country=Belgium'
+source_urls = ['https://ambermd.org/cgi-bin/AmberTools%s-get.pl' % version]
+sources = [{
+    'download_filename': local_download_credentials,
+    'filename': SOURCE_TAR_BZ2,
+}]
 patches = [
     'AmberTools-20_fix_missing_MPI_LIBRARY_error.patch',
     'AmberTools-20_cmake-locate-netcdf.patch',
     'AmberTools-21_CMake-FlexiBLAS.patch',
```

```suggestion
    'AmberTools-21_CMake-FlexiBLAS.patch',
    'AmberTools-20_fix_xblas_missing_make_dependency.patch',
```

The `xblas` patch is missing from this easyconfig.

---

There are a lot of missing patches here and I intentionally didn't add any since It was just done to verify that the tests would run. But yes, I should add all the patches....

---

All patches added from AT21.12